### DISCUSSION TITLE
Service graphs queries: copy filters and add a `sum by`

### DISCUSSION THREAD

**Discussion by yvrhdn** [grafana/grafana#47287] [Observability]
I wanted to share two points of feedback regarding the service graphs panel in Tempo, more specifically the queries that are opened in the side panel.

#### 1. Filters aren't copied over when you open a query in the side panel

For example on the screenshot I added a filter `source="tempo"`, I expect to see a subset of my services. When I open the request rate in a new panel, I would expect to have the same filters on the PromQL query.
I can imagine people will be using these filters to see parts of their services using filters like `namespace="tempo-dev"` and `service=~"tmepo-.*"`. When they open a side panel, I'd argue these metrics should be scoped the same way.

#### 2. When I open the (failed) request rate in explore I feel like I’m missing a `sum by (client, server)`

If you don’t sum the metrics you see all the different instances generating the metrics which is distracting (see the screenshot). If you add `sum by (client, server)` you only get a couple of series back, which is much cleaner.
I think a `sum without (pod, instance)` might also work, but in that case we would have to know in advance which labels to drop. This will be user-dependent.

![Screenshot 2022-04-04 at 19 18 01](https://user-images.githubusercontent.com/7748404/161597707-22a08fb2-7eab-4125-beeb-6d81f78bd047.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
